src/block_cache/block_mgr.c

```diff
@@ -157,11 +157,13 @@ __bm_sync_tiered_handles(WT_BM *bm, WT_SESSION_IMPL *session)
  *     Write a buffer into a block, creating a checkpoint.
```

Comment:
I don't know where to hang this comment, but I dislike that the block manager interface has ended up in the block_cache directory. It caused me a bunch of confusion while working on this.

That's probably a problem for another day, but if you agree, I'd appreciate if you can create a ticket to follow up.

Reply:
Ack -- I think Luke P looked at it as part of the initial modularity work and concluded it didn't make sense at the time. Now that we have two block managers, and we've taught the cache about deltas, I agree this is worth revisiting. Made https://jira.mongodb.org/browse/WT-14642